app/controllers/exhibits_controller.rb

```diff
@@ -1,6 +1,28 @@
 class ExhibitsController < Spotlight::ExhibitsController
   prepend_before_action :find_exhibit
 
+  def create
+    @exhibit.attributes = exhibit_params
+
+    if params[:exhibit].key?("slug") && @exhibit.save
+      @exhibit.roles.create user: current_user, role: 'admin' if current_user
+      ingest_members
```

Comment:
I don't think we can load the members before the exhibit is saved, because we need the exhibit id.  But I can move ingest_members to an after_action callback -- since it's destined to be a background job, that probably a good move anyway.